pandas/core/computation/pytables.py

```diff
@@ -233,7 +237,12 @@ def stringify(value):
                 result = metadata.searchsorted(v, side="left")
             return TermValue(result, result, "integer")
         elif kind == "integer":
-            v = int(float(v))
+            try:
+                v_dec = Decimal(v)
+            except InvalidOperation:
```

Comment:
Could you add a comment where why `float(v)` is called?

Reply:
added comments for reason and referencing to PR.